understanding/21/non-text-contrast.html

```diff
@@ -28,6 +28,18 @@ <h2>Intent</h2>
 				<p>The 3:1 contrast ratios referenced in this Success Criterion is intended to be treated as threshold values. When comparing the computed contrast ratio to the Success Criterion ratio, the computed values should not be rounded (e.g. 2.999:1 would not meet the 3:1 threshold).</p>
 			</div>
 
+			<div class="note">
+				<p>Because authors do not have control over user settings for font smoothing/anti-aliasing, when evaluating this
```

Comment:
Just clarifying the meaning of the `/` slash, for easier reading. 
Is it meant to represent 'and', 'nor', or 'or'? In this case I think it doesn't matter too much, but 'and' is the most accurate, as the sentence implies an inclusive list of things that authors cannot control.

```suggestion
				<p>Because authors do not have control over user settings for font smoothing and anti-aliasing, when evaluating this
```

Reply:
it's more that they're different terms for mostly the same thing ... some people with talk about font smoothing, others about anti-aliasing, but in the end it's all the same concept

Reply:
Ahh I see @patrickhlauke makes sense. I think in that case either 'and' and 'or' still work just fine. The one that won't work in that case is 'nor'.

They're 2 ways to refer to 1 concept that can't be controlled by an author, but generally they are interchangeable.

Reply:
to be difficult, it could even be "and/or" as certain OSs/browsers/users may refer to these interchangeably, or actually see them as two different ways to achieve the same thing... which is why i opted for the slash. happy to change it to either "and" or "or" though, if that makes it easier...